app/models/manageiq/providers/openstack/cloud_manager/vm/operations/relocation.rb

```diff
@@ -0,0 +1,26 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::Operations::Relocation
+  def raw_live_migrate(options = nil)
```

Comment:
how about `options = {}`?   then the `.try` calls below can just be hash lookups